readme.md

```diff
@@ -242,11 +242,11 @@ const instance = got.extend({
 ###### responseType
 
 Type: `string`<br>
-Default: `text`
+Default: `'default'`
```

Comment:
You should probably explain what `default` actually means.

Reply:
will do